File: quizCSharp/quizCSharp/viseo.quiz/DNode.cs
using System;

namespace QuizCSharp.viseo.quiz
{
    class DNode<E>
    {
        public E Element { get; set; }

        public DNode<E> Previous { get; set; }
        public DNode<E> Next { get; set; }

        public DNode(E element, DNode<E> previous, DNode<E> next)
        {
            this.Element = element;

            this.Previous = previous;
            this.Next = next;
        }

        public DNode()
        {
        }

        public override String ToString()
        {
            return null == Element ? null : Element.ToString();
        }

    }
}

File: quizJava/src/main/java/viseo/quiz/DNode.java
package viseo.quiz;

public class DNode<E> {

    private E element;

    private DNode<E> previous;
    private DNode<E> next;

    /**
     * @param element
     * @param previous
     * @param next
     */
    public DNode(E element, DNode<E> previous, DNode<E> next) {
        this.element = element;

        this.previous = previous;
        this.next = next;
    }

    @Override
    public String toString() {
        return null == element ? null : element.toString();
    }

    public E getElement() {
        return element;
    }

    public DNode<E> getPrevious() {
        return previous;
    }

    public void setPrevious(DNode<E> previous) {
        this.previous = previous;
    }

    public DNode<E> getNext() {
        return next;
    }

    public void setNext(DNode<E> next) {
        this.next = next;
    }

}
